e2e/test.py

```diff
@@ -142,6 +142,9 @@ def emit(self, record):
     logger.fatal('Expected key count of %d (got %d)', REPO_KEYS_LEN_EXPECTED,
                  REPO_KEYS_LEN_ACTUAL)
 
+logger.info('repository config: %s', restic.cat.config())
+logger.info('repository masterkey: %s', restic.cat.masterkey())
+
```

Comment:
Can we exercise all the `cat` APIs in the e2e test? I don't understand how the client obtains any of the IDs for `cat.pack`, `cat.index`, etc.

Reply:
That would require me to implement `restic list` as well which I'd like to avoid. I only need `restic cat config` to test if a repository is available with the defined repository location/password.

Reply:
Can we reduce this PR to just `cat.config` and `cat.masterkey` and forget about the rest? If there's no way to practically use them until we implement `restic list`, it doesn't really make sense to include them now.

Reply:
I'm not sure if this is a good idea. The work is already done for all subcommands, the only thing missing are E2E tests. People might use those `cat` commands with input from external sources which don't require the Python tooling to run a `list`.

E2E tests for the missing `cat` commands could simply be implemented when $someone implements the `list` command.

Reply:
There's a maintenance cost, though. I don't really want to maintain code that I can't exercise in the e2e tests to verify everything's working.

Reply:
kk, done :)